app/Http/Controllers/Assets/AssetCheckoutController.php

```diff
@@ -71,6 +71,10 @@ public function store(AssetCheckoutRequest $request, $assetId) : RedirectRespons
 
             $admin = auth()->user();
 
+            if($request->filled('audit_on_checkinout') == "1") {
+                $this->authorize('audit', Asset::class);
```

Comment:
This will probably result in a forbidden error, which is probably not what we want. We probably want to quietly ignore if they don't have permission to audit.

Reply:
okay yea. i see how that could happen.

Reply:
unsure how to fix this. originally thought was if the checkbox is filled, then check if they can do it, but don't check if its not a checked box. i am stumped on reversing that logic. may just need to write it out on my whiteboard. reverse causality and such.

Reply:
i think i did this already while i was mulling over how to fix it. if you could double check the more recent changes?